Sentry.xcodeproj/project.pbxproj

```diff
@@ -5261,7 +5265,7 @@
 				DEVELOPMENT_TEAM = "";
 				EXECUTABLE_PREFIX = lib;
 				GCC_C_LANGUAGE_STANDARD = gnu99;
-				MACH_O_TYPE = staticlib;
+				MACH_O_TYPE = mh_dylib;
```

Comment:
Please revert these, this is not a dynamic library. I can't remember what it was anymore, but something was previously broken due to it being mistyped, until I changed it to `staticlib`.

Reply:
This is SentryTestUtils and we need it to be dynamic to test our integration with dynamic libs. 
This is just an auxiliary lib, no reason it could not be dynamic lib.

Reply:
I just remembered what this was breaking, it causes the tvOS unit tests to crash when I run it locally: 
```
2023-11-28 11:21:45.982597-0900 xctest[15149:9584462] [general] Error loading /Users/andrewmcknight/Library/Developer/Xcode/DerivedData/Sentry-gnnqqmcfqpqmmhcjxfzwadgjajnt/Build/Products/Test-appletvsimulator/SentryProfilerTests.xctest/SentryProfilerTests (177):  dlopen(/Users/andrewmcknight/Library/Developer/Xcode/DerivedData/Sentry-gnnqqmcfqpqmmhcjxfzwadgjajnt/Build/Products/Test-appletvsimulator/SentryProfilerTests.xctest/SentryProfilerTests, 0x0109): Library not loaded: /Users/andrewmcknight/Library/Developer/Xcode/DerivedData/Sentry-gnnqqmcfqpqmmhcjxfzwadgjajnt/Build/Products/Test-appletvsimulator/libSentryTestUtils.a
  Referenced from: <2D742D1E-D6A4-37DB-88B5-1F09B59C9473> /Users/andrewmcknight/Library/Developer/Xcode/DerivedData/Sentry-gnnqqmcfqpqmmhcjxfzwadgjajnt/Build/Products/Test-appletvsimulator/SentryProfilerTests.xctest/SentryProfilerTests
  Reason: tried: '/Users/andrewmcknight/Library/Developer/Xcode/DerivedData/Sentry-gnnqqmcfqpqmmhcjxfzwadgjajnt/Build/Products/Test-appletvsimulator/libSentryTestUtils.a' (code signature in <F4A23AD7-1604-3FA6-94F5-463FFC70C956> '/Users/andrewmcknight/Library/Developer/Xcode/DerivedData/Sentry-gnnqqmcfqpqmmhcjxfzwadgjajnt/Build/Products/Test-appletvsimulator/libSentryTestUtils.a' not valid for use in process: Trying to load an unsigned library), '/Applications/Xcode15/Xcode.app/Contents/Developer/Platforms/AppleTVSimulator.platform/Developer/usr/lib/libSentryTestUtils.a' (no such file), '/Library/Developer/CoreSimulator/Volumes/tvOS_21J353/Library/Developer/CoreSimulator/Profiles/Runtimes/tvOS 17.0.simruntime/Contents/Resources/RuntimeRoot/Users/andrewmcknight/Library/Developer/Xcode/DerivedData/Sentry-gnnqqmcfqpqmmhcjxfzwadgjajnt/Build/Products/Test-appletvsimulator/libSentryTestUtils.a' (no such file), '/Users/andrewmcknight/Library/Developer/Xcode/DerivedData/Sentry-gnnqqmcfqpqmmhcjxfzwadgjajnt/Build/Products/Test-appletvsimulator/libSentryTestUtils.a' (code signature in <F4A23AD7-1604-3FA6-94F5-463FFC70C956> '/Users/andrewmcknight/Library/Developer/Xcode/DerivedData/Sentry-gnnqqmcfqpqmmhcjxfzwadgjajnt/Build/Products/Test-appletvsimulator/libSentryTestUtils.a' not valid for use in process: Trying to load an unsigned library), '/Library/Developer/CoreSimulator/Volumes/tvOS_21J353/Library/Developer/CoreSimulator/Profiles/Runtimes/tvOS 17.0.simruntime/Contents/Resources/RuntimeRoot/usr/lib/libSentryTestUtils.a' (no such file)
2023-11-28 11:21:46.265470-0900 xctest[15149:9584462] [Default] The bundle “SentryProfilerTests.xctest” couldn’t be loaded. Try reinstalling the bundle.
2023-11-28 11:21:46.265640-0900 xctest[15149:9584462] [Default] (dlopen(/Users/andrewmcknight/Library/Developer/Xcode/DerivedData/Sentry-gnnqqmcfqpqmmhcjxfzwadgjajnt/Build/Products/Test-appletvsimulator/SentryProfilerTests.xctest/SentryProfilerTests, 0x0109): Library not loaded: /Users/andrewmcknight/Library/Developer/Xcode/DerivedData/Sentry-gnnqqmcfqpqmmhcjxfzwadgjajnt/Build/Products/Test-appletvsimulator/libSentryTestUtils.a
  Referenced from: <2D742D1E-D6A4-37DB-88B5-1F09B59C9473> /Users/andrewmcknight/Library/Developer/Xcode/DerivedData/Sentry-gnnqqmcfqpqmmhcjxfzwadgjajnt/Build/Products/Test-appletvsimulator/SentryProfilerTests.xctest/SentryProfilerTests
  Reason: tried: '/Users/andrewmcknight/Library/Developer/Xcode/DerivedData/Sentry-gnnqqmcfqpqmmhcjxfzwadgjajnt/Build/Products/Test-appletvsimulator/libSentryTestUtils.a' (code signature in <F4A23AD7-1604-3FA6-94F5-463FFC70C956> '/Users/andrewmcknight/Library/Developer/Xcode/DerivedData/Sentry-gnnqqmcfqpqmmhcjxfzwadgjajnt/Build/Products/Test-appletvsimulator/libSentryTestUtils.a' not valid for use in process: Trying to load an unsigned library), '/Applications/Xcode15/Xcode.app/Contents/Developer/Platforms/AppleTVSimulator.platform/Developer/usr/lib/libSentryTestUtils.a' (no such file), '/Library/Developer/CoreSimulator/Volumes/tvOS_21J353/Library/Developer/CoreSimulator/Profiles/Runtimes/tvOS 17.0.simruntime/Contents/Resources/RuntimeRoot/Users/andrewmcknight/Library/Developer/Xcode/DerivedData/Sentry-gnnqqmcfqpqmmhcjxfzwadgjajnt/Build/Products/Test-appletvsimulator/libSentryTestUtils.a' (no such file), '/Users/andrewmcknight/Library/Developer/Xcode/DerivedData/Sentry-gnnqqmcfqpqmmhcjxfzwadgjajnt/Build/Products/Test-appletvsimulator/libSentryTestUtils.a' (code signature in <F4A23AD7-1604-3FA6-94F5-463FFC70C956> '/Users/andrewmcknight/Library/Developer/Xcode/DerivedData/Sentry-gnnqqmcfqpqmmhcjxfzwadgjajnt/Build/Products/Test-appletvsimulator/libSentryTestUtils.a' not valid for use in process: Trying to load an unsigned library), '/Library/Developer/CoreSimulator/Volumes/tvOS_21J353/Library/Developer/CoreSimulator/Profiles/Runtimes/tvOS 17.0.simruntime/Contents/Resources/RuntimeRoot/usr/lib/libSentryTestUtils.a' (no such file))
Program ended with exit code: 83
``` 

this is then fixed when I change it back to a static lib. There are actual differences between static and dynamic libs, they are not interchangeable.

Reply:
Damn Apple!! Why does this only fail for tvOS?
Why they need a signed lib to run for tests. Such BS.

Reply:
Yep, mysterious to me as well as to why it's like this. Sorry it was an extra pain.